thefuck/rules/cp_omitting_directory.py

```diff
@@ -0,0 +1,10 @@
+def match(command, settings):
+    if 'cp: omitting directory' in command.stderr.lower():
+        return True
+    return False
+
+
+def get_new_command(command, settings):
+    return command.script.replace('cp', 'cp -r') 
```

Comment:
This is dangerous in the case the directory name contains the letters `cp`, they will be replaced as well. Either add the `1` parameter as a third parameter (`maxreplace`), or use a regular expression, e.g.

    return re.sub('^cp (.*), 'cp -r \\1')